commands/link/__tests__/__fixtures__/with-contents/packages/package-3/cli1.js

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+console.log("Hello, world!");
```

Comment:
To make this more interesting, you might want to move this (and other bin targets) into a source location that makes sense for packages that would use `--contents`/`publishConfig.directory`. Then the snapshots would be more useful.